### PR TITLE
Style the PR creation and diff view

### PR DESCRIPTION
**Pull request by txsmith**
screenshot from 2016-05-17 13 55 24

screenshot from 2016-05-17 13 56 07

### CODE REVIEW

File: src/projects/create-pull-request/project-create-pull-request-description.html [hunk 1]
```diff
@@ -32,12 +32,13 @@
...
-      owner="[[owner]]"
-      name="[[name]]"
-      markdown="{{description}}"></markdown-input>
+      <paper-input class="title" label="Enter a pull request title" value="{{title}}"></paper-input>
+      <markdown-input
+        label="Enter a description for your pull request. Markdown is supported."
+        owner="[[owner]]"
+        name="[[name]]"
+        markdown="{{description}}"></markdown-input>
+    </div>
```
Comment (TimvdLippe) [inline src/projects/create-pull-request/project-create-pull-request-description.html]: This div should be removed I think? Then the indentation must be reverted too.

File: src/projects/create-pull-request/project-create-pull-request.html [hunk 2]
```diff
@@ -48,31 +57,29 @@
...
+      </div>
 
-    <project-create-pull-request-description
-      id="description"
-      owner="[[owner]]"
-      name="[[name]]"
-      title$="[[title]]"
-      description="[[description]]"
-      ></project-create-pull-request-description>
+      <project-create-pull-request-description
```
Comment (TimvdLippe) [inline src/projects/create-pull-request/project-create-pull-request.html:75]: Indentation should be 1 tab to the left.

Comment (txsmith) [inline src/projects/create-pull-request/project-create-pull-request.html:75]: Is solved

**Comment by TimvdLippe** [comment]
Lovely, this fixes some of the issues as described in https://github.com/preview-code/rite-evaluation/pull/64#issuecomment-219308437 Thanks!

**Pull request merged**